src/GitHub.VisualStudio/Services/UsageTracker.cs

```diff
@@ -156,25 +148,12 @@ void SaveUsage(UsageStore store)
             writeAllText(storePath, json, Encoding.UTF8);
         }
 
-        void UpdateTimer(bool initialCall)
+        void RunTimer()
         {
-            // If the method was called due to userSettings.CollectMetrics changing, then send an
-            // opt-in/out message.
-            if (!initialCall && client != null)
-            {
-                if (userSettings.CollectMetrics)
-                    client.SendOptIn();
-                else
-                    client.SendOptOut();
-            }
-
-            // The timer first ticks after 1 minute to allow things to settle down after startup.
+            // The timer first ticks after 3 minutes to allow things to settle down after startup.
             // This will be changed to 8 hours after the first tick by the TimerTick method.
-            timer.Stop();
-            timer.Interval = TimeSpan.FromMinutes(1);
-
-            if (userSettings.CollectMetrics && client != null)
-                timer.Start();
+            timer.Interval = TimeSpan.FromMinutes(3);
+            timer.Start();
         }
 
         async void TimerTick(object sender, EventArgs e)
```

Comment:
`async void` methods will crash the process if they ever throw an exception. Please wrap your entire method in a try/catch block and swallow the exception (after reporting the exception some other way like telemetry or an error dialog) to prevent this.